stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -1803,6 +1800,29 @@ impl NakamotoChainState {
         ))
     }
 
+    /// Get the aggregate public key for a block
```

Comment:
Maybe just expand this comment that this is fetching the _expected_ stacker aggregate key for a given block.